ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2362,36 +2364,46 @@ private enum UnityLayers {
     reset_rsas_target_ = true;
   }
 
-  private static void SetIntegrators(IntPtr plugin,
-                                     ConfigNode gravity_model_config) {
-   if (gravity_model_config == null) {
+  private static void InitializeIntegrators(
+      IntPtr plugin,
+      ConfigNode numerics_parameters_config) {
+   if (numerics_parameters_config == null) {
```

Comment:
The words "parameters config" read a bit funny.  I wonder if `principia_numerics_parameterization` would work better.

Reply:
As discussed, `principia_numerics_blueprint`.